config.yaml

```diff
@@ -4,31 +4,9 @@
 options:
   log-level:
     description: |
-      Configures the log level. 
-
-      Acceptable values are: "info", "debug", "warning", "error" and "critical"
+      Configures the log level.
```

Comment:
**todo** Remove the log-level option

I did not pay attention to this before, but in general logging is handled centrally by juju, not a charm configuration level. This was one can change logging on the fly, with no need to change the status of the application

Reply:
Workload log level configuration under discussion [here](https://chat.charmhub.io/charmhub/pl/jhxs3sgjbtr9xkust6qcj566co).